src/Web.cpp

```diff
@@ -925,39 +925,56 @@ void explorerHandleListRequest(AsyncWebServerRequest *request) {
 		return;
 	}
 
-	File file = root.openNextFile();
+	#if defined(HAS_FILEEXPLORER_SPEEDUP) || (ESP_ARDUINO_VERSION_MAJOR == 2 && ESP_ARDUINO_VERSION_MINOR == 0 && ESP_ARDUINO_VERSION_PATCH >= 8)
```

Comment:
Also here, if you want to use the combined macro (it should be consistently used)
```suggestion
	#if defined(HAS_FILEEXPLORER_SPEEDUP) || ESP_ARDUINO_VERSION >= ESP_ARDUINO_VERSION_VAL(2, 0, 8)
```